src/queries/voting-data.ts

```diff
@@ -10,8 +10,9 @@ export interface VotingData {
   snapshotBlockNumber: number;
 }
 
+console.log(process.env.CHAIN_ID);
```

Comment:
Was this left here by accident? If not, it probably needs some more description.

Reply:
Used for debugging - now removed